src/tests/test_decorators.py

```diff
@@ -0,0 +1,39 @@
+from . import base
+from mock import patch
+import decorators
+
+class TestDecorators(base.BaseCase):
+    def test_timeit_smoke_test(self):
+        @decorators.timeit
+        def some_task():
+            pass
+        some_task()
```

Comment:
sure, I'm all for boosting coverage, but I think the logic should be tested at the same time, otherwise it will be forgotten about and the high coverage will give us a false sense of security.